api/root_view.py

```
@@ -0,0 +1,39 @@
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+
+
+@api_view(['GET'])
+@permission_classes([AllowAny])
+def api_root(request):
```

Ruff (0.12.2)

Missing return type annotation for public function `api_root`

(ANN201)